mkosi/distributions/gentoo.py

```diff
@@ -164,13 +164,15 @@ def __init__(self, state: MkosiState) -> None:
             "--nospinner",
         ]
         if "build-script" in ARG_DEBUG:
-            self.emerge_default_opts += ["--verbose", "--quiet=n", "--quiet-fail=n"]
+            self.emerge_default_opts += ["--verbose", "--quiet=n",
+                                         "--quiet-fail=n"]
         else:
             self.emerge_default_opts += ["--quiet-build", "--quiet"]
 
         self.arch, _ = ARCHITECTURES[state.config.architecture or "x86_64"]
         self.arch_profile = Path(f"default/linux/{self.arch}/"
-                                 f"{state.config.release}/no-multilib/systemd/merged-usr")
+                                 f"{state.config.release}/"
+                                 "no-multilib/systemd/merged-usr")
```

Comment:
Any need to change these?

Reply:
> Any need to change these?

E501 line too long (size > 79 characters)

Reply:
What did you run to get that warning and how did you run it? We set 
```
[flake8]
max-line-length = 119
```
in `setup.cfg`

Reply:
No but it makes much more sense for this to be one line instead of split over multiple lines so let's keep it that way. Maybe figure out how to configure the line length for pyright/pylsp in nvim?